controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -325,7 +358,11 @@ func (r *StorageClassRequestReconciler) reconcilePhases() (reconcile.Result, err
 				return reconcile.Result{}, err
 			}
 
-			if err := r.reconcileCephBlockPool(&storageProfile); err != nil {
+			if err := r.reconcileCephBlockPool(); err != nil {
+				return reconcile.Result{}, err
+			}
```

Comment:
This call can be removed, this controller should not reconcile any block pools.  
The only thing we might want to keep is this line (from the `reconcileCephBlockPool`) func
```
r.setCephResourceStatus(r.cephBlockPool.Name, "CephBlockPool", phase, nil)
```

Reply:
Not gonna bother.